src/hardware/adc.rs

```diff
@@ -114,8 +114,8 @@ impl Adc {
         // set sync low first for synchronization at rising edge
         pins.sync.set_low().unwrap();
 
-        // SPI at 1 MHz. SPI MODE_0: idle low, capture on first transition
-        let spi: Spi<_, _, u8> = spi4.spi(pins.spi, spi::MODE_0, 12500.khz(), spi4_rec, clocks);
+        // SPI at 1 MHz. SPI MODE_0: idle high, capture on second transition
```

Comment:
The comment is now wrong in two places.
DRY and do one thing at a time.

Reply:
Sorry, I got distracted because I saw invalid data from the adc and then forgot.